BaseTools/Plugin/OverrideValidation/OverrideValidation.py

```diff
@@ -275,7 +285,9 @@ def override_process_line(self, thebuilder, overridecnt, filepath, filelist, mod
                 m_node.status = result
                 return result
 
-            if version_match[0] == 1:
+            if version_match[0] == 1 and tagtype == 'deprecated':

```

Comment:
```suggestion
            if version_match[0] == 1 and tagtype == 'deprecated':
            if version_match[0] == 1:
                if tagtype == 'deprecated':
```
then you can remove the elif?